README.md

```diff
@@ -82,6 +82,7 @@ func main() {
 | [ST7735 TFT color display](https://www.crystalfontz.com/controllers/Sitronix/ST7735R/319/) | SPI |
 | [ST7789 TFT color display](https://cdn-shop.adafruit.com/product-files/3787/3787_tft_QT154H2201__________20190228182902.pdf) | SPI |
 | [Thermistor](https://www.farnell.com/datasheets/33552.pdf) | ADC |
+| [VEML6070](https://www.vishay.com/docs/84277/veml6070.pdf) | I2C |
```

Comment:
This should probably say "VEML6070 UV light sensor" not just "VEML6070".

Reply:
You're right, done! Anything else?